lib/zoom_pinch_overlay.dart

```diff
@@ -55,13 +55,17 @@ class ZoomOverlay extends StatefulWidget {
   /// Specifies the animation duartion when the widget zoom has ended and is animating back to the original place.
   final Duration animationDuration;
 
+  /// Specifies the animation curve when the widget zoom has ended and is animating back to the original place.
+  final Curve animationCurve;
+
   const ZoomOverlay(
       {Key? key,
       this.twoTouchOnly = false,
       required this.child,
       this.minScale,
       this.maxScale,
-      this.animationDuration = const Duration(milliseconds: 100)})
+      this.animationDuration = const Duration(milliseconds: 100),
+      this.animationCurve})
```

Comment:
This has to have a default value.
`this.animationCurve = Curves.linear`

Reply:
🙊 I was not thinking about it at all!